change/@ni-nimble-components-95871031-15d3-48ad-952f-7c70831fb3d0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Updated the WaferDie Interface to add an extra \"tooltip\" field",
+  "packageName": "@ni/nimble-components",
+  "email": "denis.stavila@ni.com",
```

Comment:
Note that it's using your NI email instead of your github one. You can set git user per repo if needed: https://stackoverflow.com/questions/18181439/setting-different-config-for-different-repositories

Reply:
I keep forgetting that :) ... applied